package.json

```diff
@@ -128,7 +128,7 @@
     "react-dom": "^16.13.1",
     "react-hotkeys": "^1.1.4",
     "react-immutable-proptypes": "^2.2.0",
-    "react-immutable-pure-component": "^1.1.1",
+    "react-immutable-pure-component": "^2.2.2",
```

Comment:
> Version `v2.0.1` introduced minimal `immutable` version to >= 4, with this release this limitation is removed. Now everything will work exactly the same way in `react-immutable-pure-component@1` with `immutable@<4`. In case when value other then `string` is in `updateOnProps` or `updateOnStates` the `TypeError` will be thrown.